core/merginapi.cpp

```diff
@@ -3823,6 +3847,55 @@ void MerginApi::deleteAccountFinished()
   r->deleteLater();
 }
 
+void MerginApi::updateWorkspaceStorageProjectLimit( const QString &workspaceId, const int storageLimit, const int projectLimit )
+{
+  if ( !validateAuth() || mApiVersionStatus != MerginApiStatus::OK )
+  {
+    return;
+  }
+
+  QNetworkRequest request = getDefaultRequest();
+  const QUrl url( mApiRoot + QStringLiteral( "/v1/tests/workspaces/%1" ).arg( workspaceId ) );
+  request.setUrl( url );
+  request.setHeader( QNetworkRequest::ContentTypeHeader, "application/json" );
+
+  // Create JSON payload
+  QJsonObject limitsOverride;
+  limitsOverride["storage"] = storageLimit;
+  limitsOverride["projects"] = projectLimit;
+  limitsOverride["api_allowed"] = true;
+
+  QJsonObject payload;
+  payload["limits_override"] = limitsOverride;
+
+  const QJsonDocument doc( payload );
+  const QByteArray data = doc.toJson();
+
+  // Send PATCH request
+  QNetworkReply *reply = mManager->sendCustomRequest( request, "PATCH", data );
+
+  connect( reply, &QNetworkReply::finished, this, &MerginApi::updateWorkspaceStorageLimitFinished );
+
+  CoreUtils::log( "update workspace storage limit",
+                  QStringLiteral( "Updating workspace: " ) + url.toString() );
```

Comment:
```suggestion
  CoreUtils::log( QStringLiteral( "update workspace storage limit" ),
                  QStringLiteral( "Updating workspace: " ) + url.toString() );
```
you could also use `.arg()` here instead of concatenation